src/components/MemberDescription.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MemberDescription = ({
+    name,
+    role,
+    img,
+    education,
+    companies,
+    bio,
+    speaking,
+}) => {
+    return (
+        <div className="overflow-hidden bg-white">
+            <div className="relative mx-auto max-w-6xl py-14 px-4 sm:px-6 lg:px-8">
+                <div className="mt-7 lg:grid gap-0 lg:grid-cols-2">
+                    <div className="relative ml-8 mt-4 lg:col-start-1 lg:row-start-1">
+                        <div className="relative mx-auto max-w-prose text-base lg:max-w-none">
+                            <figure>
+                                <img
```

Comment:
yeah I think once we get an idea of how images will be handed to us via the API we are going to build, then we can worry about the sizing but this is good for now 👍